### ISSUE TITLE
RuntimeError: CameraInit failed with error code 1 on every Picture I try.

### ISSUE THREAD

**Issue by LercheSi** [alicevision/meshroom#1431]
Describe the bug
Images cannot be imported.

To Reproduce
Drag and drop image files into meshroom or use the import menu.

Following error message comes up using drag and drop:
Failed to Import Images - Meshroom 2021.1.0

You probably have a corrupted image within the images that you are trying to import.

Screenshots

Log
Following error message is shown in the console using the import menu.

If applicable, copy paste the relevant log output (please embed the text in a markdown code tag "\`\`\`" )
`Program called with the following parameters:
 * allowSingleView = 1
 * allowedCameraModels = "pinhole,radial1,radial3,brown,fisheye4,fisheye1"
 * defaultCameraModel = "" (default)
 * defaultFieldOfView = 45
 * defaultFocalLengthPix = -1 (default)
 * defaultIntrinsic = "" (default)
 * groupCameraFallback =  Unknown Type "enum EGroupCameraFallback"
 * imageFolder = "" (default)
 * input = "C:\Users\Simon\AppData\Local\Temp\tmpapei8tx6/CameraInit/f9436e97e444fa71a05aa5cf7639b206df8ba282//viewpoints.sfm"
 * output = "C:/Users/Simon/AppData/Local/Temp/tmpapei8tx6/CameraInit/f9436e97e444fa71a05aa5cf7639b206df8ba282/cameraInit.sfm"
 * sensorDatabase = "D:\Meshroom-2021.1.0-win64\Meshroom-2021.1.0\aliceVision\share\aliceVision\cameraSensors.db"
 * useInternalWhiteBalance = 1
 * verboseLevel = "info"
 * viewIdMethod =  Unknown Type "enum aliceVision::sfmDataIO::EViewIdMethod"
 * viewIdRegex = ".*?(\d+)" (default)

[08:59:18.776991][info] CameraInit report:
        - # views listed: 1
           - # views with an initialized intrinsic listed: 1
           - # views without metadata (with a default intrinsic): 0
        - # intrinsics listed: 1
ERROR:root:Error while building intrinsics: CameraInit failed with error code 1.
Command was: "aliceVision_cameraInit  --sensorDatabase "D:\Meshroom-2021.1.0-win64\Meshroom-2021.1.0\aliceVision\share\aliceVision\cameraSensors.db" --defaultFieldOfView 45.0 --groupCameraFallback folder --allowedCameraModels pinhole,radial1,radial3,brown,fisheye4,fisheye1 --useInternalWhiteBalance True --viewIdMethod metadata --verboseLevel info --output "C:/Users/Simon/AppData/Local/Temp/tmpapei8tx6/CameraInit/f9436e97e444fa71a05aa5cf7639b206df8ba282/cameraInit.sfm" --allowSingleView 1 --input "C:\Users\Simon\AppData\Local\Temp\tmpapei8tx6/CameraInit/f9436e97e444fa71a05aa5cf7639b206df8ba282//viewpoints.sfm"".

WARNING:root:QQmlComponent: Created graphical object was not placed in the graphics scene.
Desktop (please complete the following and other pertinent information):`

OS: Win 10
Qt/PySide version not installed
Meshroom version: Meshroom-2021.1.0
Binary version (if applicable) [e.g. 2020.1.1]
Additional context:
Pictures made with Nikon D5600

Failed attempts of solving the issue

reinstalling Meshroom
reinstalling Microsoft Visual C++ Redistributable for Visual Studio 2015, 2017 and 2019
checked Images with "Bad Peggy"

**Comment by jonboy73** [comment]
i am having the same problem and i have only recently downloaded the software. i am using a Nikon D5100 so is it the camera or software issue. i do not know about programming and only starting to try photogrammetry as enjoy doing 3D printing and trying to look for more projects.

**Comment by natowi** [comment]
What happens when you batch convert your images with (IrfranView) and remove the Exif information? Does it work?

**Comment by chameroy** [comment]
I am having the same issue with 2023.1.0 on Windows 11.

Here is the output I'm receiving:

ERROR:root:Error while building intrinsics: CameraInit failed with error code 1.
Command was: "aliceVision_cameraInit  --sensorDatabase "C:\Users\Chameroy\Desktop\Meshroom-2023.1.0\aliceVision\share\aliceVision\cameraSensors.db" --lensCorrectionProfileInfo "${ALICEVISION_LENS_PROFILE_INFO}" --lensCorrectionProfileSearchIgnoreCameraModel True --defaultFieldOfView 45.0 --groupCameraFallback folder --allowedCameraModels pinhole,radial1,radial3,brown,fisheye4,fisheye1,3deanamorphic4,3deradial4,3declassicld --rawColorInterpretation LibRawWhiteBalancing --viewIdMethod metadata --verboseLevel info --output "C:/Users/Chameroy/AppData/Local/Temp/tmpg3ziu8w6/CameraInit/ee5aca9875d6756077ac43f2ac3f4b6023cde933/cameraInit.sfm" --allowSingleView 1 --input "C:\Users\Chameroy\AppData\Local\Temp\tmpg3ziu8w6/CameraInit/ee5aca9875d6756077ac43f2ac3f4b6023cde933/viewpoints.sfm"".

WARNING:root:QQmlComponent: Created graphical object was not placed in the graphics scene.

This happens even when attempting to import the monstree image set.

I've also tried reinstalling the MS VC++ Redistributables and had no change.

I've tested this on another machine running Windows 11 and had no issues.
